boot/boot.ks

```diff
@@ -65,43 +100,43 @@ ELSE {
 			REBOOT.
 		}
 		ELSE {
-			PRINT "Career insufficiently advanced to deploy antennas -- aborting reboot!".
+			bootError "Cannot contact KSC. Does your ship have enough fixed antennas?".
```

Comment:
Would it be excessive to check for fixed antennas, and only if none are found run the CANDOACTIONS test before trying to deploy antennas?  Right now, I think the `bootError` line will be printed if there are fixed antennas but the VAB is not upgraded.

Reply:
I thought about it, but the user's required course of action is the same in either case: he needs to Add Moar Antenna, and if he can't do actions, then he can't deploy retractable antennas by himself. Hence the error message we show him, should nudge him to exactly the right course of action. (Let me know if my logic is flawed.)

Reply:
No, that makes perfect sense, especially for players who are trying to run the entire mission through the terminal.

Reply:
@fellipec this may be a good starting point if you end up hacking the boot code on Tuesday. Many thanks for your continued efforts!